### DISCUSSION TITLE
Article preview

### DISCUSSION THREAD

**Discussion by bluenex** [dailydotdev/daily#458] [Feature Requests]
Here is an email content I replied to the daily.dev team:

> Hello Daily.dev team,
>
> I have read more daily after using [daily.dev](http://daily.dev/), it is a really good app in a good place (on new tab). One thing that could be an improvement for the UX to me is the way to open articles. By default, clicking the article brings us to a new tab. It would be cool if it has a way to load the whole article as is (iframe) and render in a modal with a little button or shortcut keys to navigate to the original site.
>
> Don't get me wrong, opening in a new tab is good and practical. But for some articles, I think it would be cool if we can just have a glance to decide whether to follow til the end or just skip it.

And I got a reply that it is in the roadmap already. It would be cool if we can gather opinion and keep getting update in this thread.

**Comment by nimrodkra** [comment]
Hey @bluenex, thank you for following up and submitting the feature request here as well! That way, more people can engage and provide their point of view :) I just wanted to correct the status of this feature request slightly. We are considering the option to change the default click on articles to open daily.dev's Article Page ([example](https://app.daily.dev/posts/enrnNxxHl)). The big advantage for doing so is the ability to quickly read the article's TLDR before deciding if you want to read the full text or not. Opening up the full-text articles in an iframe or something similar is out of scope for now.
